bin/concurrently.ts

```diff
@@ -110,6 +110,11 @@ const args = yargs(argsBeforeSep)
             describe: 'Kill other processes if one exits with non zero status code.',
             type: 'boolean',
         },
+        'kill-signal': {
+            alias: 'ks',
+            describe: 'Signal to send to other processes if one exits or dies.',
```

Comment:
It'd be useful to list what the default is here. Take a look at `passthrough-arguments` and `timings` for some examples

Reply:
Fixed.